vNext/test/DefaultCluster.Tests/DefaultCluster.Tests.csproj

```diff
@@ -15,7 +15,11 @@
   </ItemGroup>
 
   <ItemGroup>
-    <Compile Include="..\..\..\test\DefaultCluster.Tests\**\*.cs" Exclude="..\..\..\test\DefaultCluster.Tests\FSharpGrainTests.cs;..\..\..\test\DefaultCluster.Tests\obj\**\*.cs;..\..\..\test\DefaultCluster.Tests\bin\**\*.cs;..\..\..\test\DefaultCluster.Tests\Properties\*.cs" />
```

Comment:
this goes back and forth... let's keep it without line feeds, as some editors tend to dislike them and and XML friendly line feeds.